test/metabase/models/revision/diff_test.clj

```diff
@@ -1,37 +0,0 @@
-(ns metabase.models.revision.diff-test
```

Comment:
these tests are covered in test/metabase/models/revision_test.clj , this is just a duplication